pipert/contrib/flask_display.py

```diff
@@ -40,22 +37,25 @@ def __init__(self, in_key_meta, in_key_im, url, queue, *args, **kwargs):
         self.in_key_meta = in_key_meta
         self.in_key_im = in_key_im
         self.url = url
-        self.queue = queue
+        self.q_handler = QueueHandler(queue)
         self.msg_handler = None
         self.flip = False
         self.negative = False
 
-    def receive_msg(self, in_key):
-        encoded_msg = self.msg_handler.receive(in_key)
+    def receive_msg(self, in_key, most_recent=True):
+        if most_recent:
+            encoded_msg = self.msg_handler.read_most_recent_msg(in_key)
+        else:
+            encoded_msg = self.msg_handler.receive(in_key)
         if not encoded_msg:
             return None
         msg = message_decode(encoded_msg)
         msg.record_entry(self.component_name, self.logger)
         return msg
 
     def main_logic(self, *args, **kwargs):
-        pred_msg = self.receive_msg(self.in_key_meta)
-        frame_msg = self.receive_msg(self.in_key_im)
+        pred_msg = self.receive_msg(self.in_key_meta, most_recent=False)
```

Comment:
Why False?

Reply:
Right now using the `RedisHandler` methods `read_most_recent_msg` and `read_next_msg` cause problems with the visualizations of the detections. I still don't know if it's a bug in those methods or bad implementaion on the flask display side (most likely something with flask display), so I left it like that for now.

Reply:
@MayoG is this the bug we talked about?

Reply:
I'm pretty sure it's the way vis logic is implemented in FlaskDisplay. After the detections are received it should keep drawing the same detections until new ones are received, so it uses the old RedisHandler receive which keeps pulling for the last detections in the queue.

I think it could be left like that for now, but we should start marking some components that should be moved to core along with testing, such as `flask_display`, `vid_capture`, etc.